### ISSUE TITLE
todo list

### ISSUE THREAD

**Issue by kushalkolar** [kushalkolar/rnn-go-brr#1]
- [ ] Get distribution of start & end position, fit to some distribution, draw samples from this distribution to use at the input vector for the RNN
- [ ] Create a larger training dataset, format Teena's data into 3d
- [ ] Think about how to validate that the model can generalize, maybe split the data and compute losses on a test set??
- [ ] Gridsearch with biological constraints with and without pruning
